api/v1beta1/cluster_types.go

```diff
@@ -514,6 +527,12 @@ type ClusterAvailabilityGate struct {
 	// +kubebuilder:validation:MaxLength=316
 	// +kubebuilder:validation:MinLength=1
 	ConditionType string `json:"conditionType"`
+
+	// polarity of the conditionType specified in this availabilityGate.
+	// The default is Positive.
```

Comment:
I'd suggest adding some more context, WDYT?
```suggestion
	// polarity of the conditionType specified in this availabilityGate.
	// Valid values are Positive, Negative and omitted.
	// When omitted, the default behaviour will be Positive.
	// A positive polarity means that the condition should report a true status when the copmonent is available.
	// A negative polarity means that the condition should report a false status when the component is available.
```